### PR TITLE
refactor(surveyDetails): add tabs on SurveyDetails component

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
refactor(surveyDetails): add surveyDetails tabs

### CODE REVIEW

File: src/components/DwellingVisits/index.js [hunk 1]
```diff
@@ -52,10 +50,9 @@ class DwellingVisits extends Component {
     render() {
         const {columns, dwellingVisits} = this.props;
         return (
-            <View>
-                <Title>Visitas registradas a la vivienda</Title>
+            <Fragment>
```
Comment (navarroaxel) [inline src/components/DwellingVisits/index.js]: remover el `Fragment`

File: src/components/DwellingVisits/index.js [hunk 2]
```diff
@@ -17,11 +15,11 @@ class DwellingVisits extends Component {
         match: matchParamsIdPropTypes.isRequired,
         dwellingVisits: PropTypes.arrayOf(dwellingVisitPropTypes).isRequired,
         columns: PropTypes.arrayOf(columnPropType)
-    };
+    }
 
     static defaultProps = {
         columns: null
-    };
+    }
```
Comment (navarroaxel) [inline src/components/DwellingVisits/index.js]: el `;` dejalo

**Comment by maximilianoforlenza** [comment]
listo

**Comment by maximilianoforlenza** [comment]
listo

**Pull request merged**